tsfile/src/main/java/org/apache/iotdb/tsfile/write/writer/DefaultTsFileOutput.java

```diff
@@ -62,6 +62,7 @@ public long getPosition() throws IOException {
 
   @Override
   public void close() throws IOException {
+    outputStream.getChannel().force(true);
```

Comment:
does `close()` need a `force()`?

Reply:
If you can find where force() is called within close(), I will remove this call. Otherwise, we should call it explicitly.